python/lsst/sdm/schemas/dp1.yaml

```diff
@@ -356,142 +360,144 @@ tables:
       section 4.9.11 of Bosch et al. 2018, PASJ, 70, S5 for details. Measured on z-band.
     datatype: float
   - name: z_cModelFlux
-    description: Flux from the final cmodel fit. Forced on z-band.
+    description: Flux from the final CModel fit. Forced on z-band.
     datatype: float
     ivoa:unit: nJy
   - name: z_cModelFluxErr
-    description: Flux uncertainty from the final cmodel fit. Forced on z-band.
+    description: Flux uncertainty from the final CModel fit. Forced on z-band.
     datatype: float
     ivoa:unit: nJy
   - name: z_cModelMag
     description: AB magnitude of cModelFlux. Forced on z-band.
     datatype: float
     ivoa:unit: mag
+    tap:principal: 1
   - name: z_cModelMagErr
     description: Uncertainty in magnitudes on cModelFlux. Forced on z-band.
     datatype: float
     ivoa:unit: mag
+    tap:principal: 1
   - name: z_cModelFlux_inner
     description: Flux within the fit region, with no extrapolation. Forced on z-band.
     datatype: float
     ivoa:unit: nJy
   - name: z_free_cModelFlux
-    description: Flux from the final cmodel fit. Measured on z-band.
+    description: Flux from the final CModel fit. Measured on z-band.
     datatype: float
     ivoa:unit: nJy
   - name: z_free_cModelFluxErr
-    description: Flux uncertainty from the final cmodel fit. Measured on z-band.
+    description: Flux uncertainty from the final CModel fit. Measured on z-band.
     datatype: float
     ivoa:unit: nJy
   - name: z_free_cModelFlux_inner
     description: Flux within the fit region, with no extrapolation. Measured on z-band.
     datatype: float
     ivoa:unit: nJy
   - name: z_free_cModelFlux_flag
-    description: Flag set if the final cmodel fit (or any previous fit) failed. Measured
+    description: Flag set if the final CModel fit (or any previous fit) failed. Measured
       on z-band.
     datatype: boolean
   - name: z_hsmShapeRegauss_e1
-    description: PSF-corrected shear using Hirata & Seljak (2003) regaussianization.
+    description: PSF-corrected shear using the Hirata & Seljak (2003) Regaussianization algorithm.
       Measured on z-band.
     datatype: float
   - name: z_hsmShapeRegauss_e2
-    description: PSF-corrected shear using Hirata & Seljak (2003) regaussianization.
+    description: PSF-corrected shear using the Hirata & Seljak (2003) Regaussianization algorithm.
       Measured on z-band.
     datatype: float
   - name: z_hsmShapeRegauss_sigma
-    description: PSF-corrected shear using Hirata & Seljak (2003) regaussianization.
+    description: PSF-corrected shear using the Hirata & Seljak (2003) Regaussianization algorithm.
       Measured on z-band.
     datatype: float
   - name: z_hsmShapeRegauss_flag
-    description: General failure flag, set if anything went wrong. Measured on z-band.
+    description: Flag set for any failure with the Regaussianziation shapes. Measured on z-band.
     datatype: boolean
   - name: z_hsm_moments_30
```

Comment:
Can we confirm that all the `*hsm*` columns are indeed intended to be unitless?  (I.e., they are normalized/standardized moments?)

Reply:
Yes, these are normalized to be unit less.